### PR TITLE
sig-release: Add justaugustus to Release Managers team

### PR DESCRIPTION
**Pull request by justaugustus**
I'll be doing some branch management tasks which require superpowers e.g., `branchff`

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/area release-eng
/sig release
/cc @kubernetes/release-engineering 
/assign @tpepper @calebamiles

### CODE REVIEW

**Comment by tpepper** [comment]
I'm ok with this generally, but it also feels slightly arbitrary.  There are 13 branch management associates listed at https://github.com/kubernetes/sig-release/blob/master/release-managers.md#branch-managers.  I'd like to see more of a plan around how the two Branch Managers are mentoring this large set of folks, what tasks are delegated to them, when, for how long, and what quality controls and oversight is in place.

**Comment by tpepper** [comment]
Also are all the similar-ish related cleanups you've done in test-infra (eg: https://github.com/kubernetes/test-infra/pull/13869, https://github.com/kubernetes/test-infra/pull/13882) fully/correctly documented in the https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/branch-manager.md where they're recurring tasks?  I feel like you've hit some bumps, but a quick glance shows at least some of your changes are following the handbook.  Any gaps would be good to update.

**Comment by cpanato** [comment]
I agree with @tpepper as an associate sometimes (to not say most of the times) I feel lost, also because is it hard for people that dedicate the "free" time to do that then cannot spend too much time doing some archeology in the deep kubernetes links/repos/docs...
but on the other hand, I would like to help more, and I know in the beginning the release managers will need to dedicate more time for the associates and also delegate more things that are more straightforward to do, like job configs and other things.

if we have associates i think is the help the release managers to do other things more complex and the associates can do other things

**Comment by justaugustus** [comment]
> Also are all the similar-ish related cleanups you've done in test-infra (eg: [kubernetes/test-infra#13869](https://github.com/kubernetes/test-infra/pull/13869), [kubernetes/test-infra#13882](https://github.com/kubernetes/test-infra/pull/13882)) fully/correctly documented in the https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/branch-manager.md where they're recurring tasks? I feel like you've hit some bumps, but a quick glance shows at least some of your changes are following the handbook. Any gaps would be good to update.

@tpepper -- I'm going to hazard a guess you haven't seen:
- [`1.16.0-beta.0` release cut issue](https://github.com/kubernetes/sig-release/issues/756)
- [Updates to Branch Manager handbook](https://github.com/kubernetes/sig-release/pull/757)

**Comment by justaugustus** [comment]
p.s. I don't disagree with anything said here, but I do think it is out of scope for this PR.

@tpepper or @cpanato -- please add an agenda item about this for next week's Release Engineering meeting.

**Comment by cpanato** [comment]
I just would like to off load you @justaugustus to not burn out and be free to do other things in K as well

**Comment by justaugustus** [comment]
That's very sweet, @cpanato!
Making the Release Managers group a success is probably my most important task for the next few cycles.

Doing some of this work upfront and making sure it's succintly documented is going to help me help you to help me. 😜

/pony hugs

**Comment by justaugustus** [comment]
I'm going to update the role comment on this change and then unhold, as I'm now a Branch Manager: https://github.com/kubernetes/sig-release/pull/760

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -183,6 +183,7 @@ teams:
     - feiskyer # Patch Release Team
     - hoegaarden # Patch Release Team
     - idealhack # Branch Manager
+    - justaugustus # Release Manager Associate
```
Comment (tpepper) [inline config/kubernetes/sig-release/teams.yaml]: The #comment text here appears out of date.

**Comment by justaugustus** [comment]
@tpepper -- Fun fact: If you make updates, you have to commit them and push! :man_facepalming:
Just fixed! :)

**Comment by tpepper** [comment]
To allow work to proceed
/lgtm
but I'm still worried that a SPOF if driving us.

**Pull request merged**